storages/sqlite-storage/migrations/2026-06-26-000000_add_pending_inbound_messages/up.sql

```
@@ -0,0 +1,14 @@
+-- hook confirms the message is committed, so a crash before the commit replays
+-- the message on the next connect instead of losing it. Keyed by the stanza id.
+
+CREATE TABLE pending_inbound_messages (
+    id TEXT NOT NULL,
+    message BLOB NOT NULL,
+    device_id INTEGER NOT NULL DEFAULT 1,
+    inserted_at INTEGER NOT NULL DEFAULT (strftime('%s', 'now')),
+    PRIMARY KEY (id, device_id)
+);
+
+CREATE INDEX idx_pending_inbound_inserted ON pending_inbound_messages (inserted_at, device_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)